src/shared/components/Sidebar.tsx

```diff
@@ -412,6 +414,57 @@ const Sidebar = () => {
     </>
   )
 
+  const getInventoryLinks = () => (
+    <>
+      <ListItem
+        active={splittedPath[1].includes('inventory')}
+        onClick={() => {
+          navigateTo('/inventory')
+          setExpansion('inventory')
+        }}
+        className="nav-item"
+        style={listItemStyle}
+      >
+        <Icon
+          icon={
+            splittedPath[1].includes('inventory') && expandedItem === 'inventory'
+              ? 'down-arrow'
+              : 'right-arrow'
+          }
+          style={expandibleArrow}
+        />
+        <Icon icon="lab" /> {!sidebarCollapsed && t('inventory.label')}
+      </ListItem>
+      {splittedPath[1].includes('inventory') && expandedItem === 'inventory' && (
+        <List layout="flush" className="nav flex-column">
+          {/* Need to figure out how to give permissions */}
+          {permissions.includes(Permissions.ViewInventory) && (
```

Comment:
Should this be add item instead?
```suggestion
          {permissions.includes(Permissions.AddItem) && (
```